cmd/flags/flags.go

```diff
@@ -23,18 +23,19 @@ import (
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 
-	hc "github.com/jaegertracing/jaeger/pkg/healthcheck"
+	ae "github.com/jaegertracing/jaeger/pkg/adminendpoint"
+	hc "github.com/jaegertracing/jaeger/pkg/adminendpoint/healthcheck"
 	"github.com/jaegertracing/jaeger/plugin/storage"
 )
 
 const (
-	spanStorageType     = "span-storage.type" // deprecated
-	logLevel            = "log-level"
-	configFile          = "config-file"
-	healthCheckHTTPPort = "health-check-http-port"
+	spanStorageType       = "span-storage.type" // deprecated
+	logLevel              = "log-level"
+	configFile            = "config-file"
+	adminEndpointHTTPPort = "admin-endpoint-http-port"
```

Comment:
This seem to be a breaking change. Would it be possible to make it backwards compatible? There's a `RegisterAlias()` function in `viper` that could be used for this purpose: https://github.com/spf13/viper#registering-and-using-aliases

Reply:
May be best to just handle in the same way as other deprecated flags? e.g. https://github.com/jaegertracing/jaeger/blob/master/cmd/agent/app/reporter/tchannel/flags.go#L54-L66 - so accept the flag, log warning and copy over the value.